ipaclient/install/client.py

```diff
@@ -1337,9 +1337,29 @@ def do_nsupdate(update_txt):
         ipautil.run([paths.NSUPDATE, '-g', UPDATE_FILE])
         result = True
     except CalledProcessError as e:
-        logger.debug('nsupdate failed: %s', str(e))
+        logger.debug('nsupdate (GSS-TSIG) failed: %s', str(e))
         try:
             ipautil.run([paths.NSUPDATE, UPDATE_FILE])
+            try:
+                sssd = services.service('sssd', api)
+                sssdconfig = SSSDConfig.SSSDConfig()
+                sssdconfig.import_config()
+                domains = sssdconfig.list_active_domains()
+                for name in domains:
+                    domain = sssdconfig.get_domain(name)
+                    try:
+                        provider = domain.get_option('id_provider')
+                    except SSSDConfig.NoOptionError:
+                        continue
+                    if name == api.env.domain and provider == "ipa":
+                        if domain.get_option('dyndns_update') is True:
+                            # dyndns_update is set by configure_sssd_conf()
+                            domain.set_option('dyndns_auth', 'none')
+                            sssdconfig.save_domain(domain)
```

Comment:
You can add a break here. Once the ipa domain is found there is no need to process any others.

Reply:
Done, thanks